assets/airports/kric.json

```diff
@@ -0,0 +1,9359 @@
+                ["N38.30079", "W77.07420"],
+                ["N38.30459", "W77.07513"],
+                ["N38.30845", "W77.07514"],
+                ["N38.31225", "W77.07423"],
+                ["N38.31585", "W77.07243"],
+                ["N38.31911", "W77.06980"],
+                ["N38.32193", "W77.06644"],
+                ["N38.32419", "W77.06246"],
+                ["N38.32583", "W77.05802"],
+                ["N38.32678", "W77.05326"],
+                ["N38.32701", "W77.04836"],
+                ["N38.32650", "W77.04349"],
+                ["N38.32529", "W77.03884"],
+                ["N38.32343", "W77.03466"],
+                ["N38.31547", "W77.02153"],
+                ["N38.31514", "W77.02119"],
+                ["N38.31232", "W77.01786"],
+                ["N38.30906", "W77.01526"],
+                ["N38.30547", "W77.01349"],
+                ["N38.30168", "W77.01261"],
+                ["N38.29783", "W77.01266"],
+                ["N38.29405", "W77.01363"],
+                ["N38.29049", "W77.01548"],
+                ["N38.28726", "W77.01816"],
+                ["N38.28449", "W77.02156"],
+                ["N38.28228", "W77.02556"],
+                ["N38.28071", "W77.03002"],
+                ["N38.27982", "W77.03478"],
+                ["N38.27966", "W77.03966"],
+                ["N38.28023", "W77.04449"],
+                ["N38.28151", "W77.04910"],
+                ["N38.28342", "W77.05334"],
+                ["N38.29101", "W77.06612"]
+            ]
+        },
+        {
+            "name": "R-6613A",
+            "height": "40000ft",
+            "poly": [
+                ["N38.26267", "W76.86710"],
+                ["N38.22501", "W76.77654"],
+                ["N38.16707", "W76.83330"],
+                ["N38.21676", "W76.90988"]
+            ]
+        }
+    ],
+    "runways": [
+        {
+            "name": ["2", "20"],
+            "end": [
+                ["N37d29.932348", "W77d19.779457", "158ft"],
+                ["N37d30.991123", "W77d19.461292", "165ft"]
+            ],
+            "ils": [true, false]
+        },	
+        {
+            "name": ["16", "34"],
+            "end": [
+                ["N37d30.998187", "W77d19.414595", "166ft"],
+                ["N37d29.747552", "W77d18.413082", "160ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "airways": {
+        "J191": ["HPW", "PELTO", "ILM"],
+        "V1": ["SBY", "MAGGO", "JAMIE"],
+        "V20": ["OTT", "DAILY", "JUDED", "WHINO", "COLIN", "DUPEY", "TAPPA", "SVILL", "RIC"],
+        "V38": ["GVE", "EMMIE", "ROOKY", "RIC", "NEMAN", "HCM", "JUDLU", "TERKS", "CCV"],
+        "V155": ["RDU", "WIPER", "BEXGO", "LVL"],
+        "V157": ["LVL", "MANGE", "DALTO", "MATOX", "RIC", "SVILL", "TAPPA", "DUPEY", "COLIN", "PXT"],
+        "V260": ["FKN", "WAIKS", "HPW", "RIC", "FAK"]
+    },
+    "sids": {
+        "COLIN8": {
+            "icao": "COLIN8",
+            "name": "Colin Eight",
+            "rwy": {
```

Comment:
It appears that the definition of the fixes are slightly off to what the should be, I'm not sure if it's an issue with units during the calculation (km vs nm) and/or magnetic deviation issues. In the image below I've marked in green where the fixes are currently defined, and in red the approximate position of where they should be.
![image](https://user-images.githubusercontent.com/13125311/152981093-49510fed-afb6-4015-8dfa-55f393c043c9.png)